prepare.sh

```diff
@@ -6,9 +6,9 @@ set -e # Exit on any error
 
 ## These should be audited!
 mymonero_core_cpp_url='https://github.com/ExodusMovement/mymonero-core-cpp'
-mymonero_core_cpp_hash='e96f8830bc4a1cb07985ef66f02f4d89815bb51c'
+mymonero_core_cpp_hash='4e9110545e44abd3ef47324e0c031170e0999de8'
```

Comment:
Use the commit hash of [feature branch](https://github.com/ExodusMovement/mymonero-core-cpp/pull/32). After merging this PR to the main branch, this value must be changed.